docker/models/containers.py

```diff
@@ -0,0 +1,144 @@
+from docker.errors import ImageNotFound
+from ..errors import ContainerError
+from .images import Image
+from .resource import Collection, Model
+
+
+class Container(Model):
+
+    @property
+    def name(self):
+        """
+        Returns the name of the container.
+        """
+        if self.attrs.get('Name') is not None:
+            return self.attrs['Name'].lstrip('/')
+
+    @property
+    def status(self):
+        """
+        Returns the status of the container. For example, `running`, or
+        `exited`.
+        """
+        return self.attrs['State']['Status']
+
+    def attach(self, *args, **kwargs):
```

Comment:
It's too bad that taking a look at the source code won't inform users about the parameters that are available for that method. I understand that having to maintain a list of arguments in two separate files is a hassle to maintain, just wondering if there's another solution.